quizer_game/templates/quizer_game/quiz-index.html

```diff
@@ -0,0 +1,55 @@
+<!DOCTYPE html>
+<html lang="en">
+{% load quizer_game_extras %}
+  <head> 
+    <meta charset="UTF-8">
+    <title>Quiz index</title>
+    {% load static %}
+    <link rel="stylesheet" href="{% static 'quizer_game/styles/quiz-index.css' %}">
+  </head>
+  <body>
+
+    <header id="masthead">
+      login version
```

Comment:
I think you should hide it.

`login version`

Reply:
Already fixed.